packages/server/package.json

```diff
@@ -164,12 +164,15 @@
     "http-mitm-proxy": "0.7.0",
     "https-proxy-agent": "3.0.1",
     "istanbul": "0.4.5",
+    "mocha": "7.1.0",
```

Comment:
i feel like we should just use 7.0.1 here and skip the duplicate dependency, no biggie tho